Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -2452,6 +2452,12 @@ def test_http_request_ssl_error(self, requests_mock):
         with raises(DemistoException, match="SSL Certificate Verification Failed"):
             self.client._http_request('get', 'event', resp_type='response')
 
+    def test_http_request_ssl_error_insecure(cls, requests_mock):
+        requests_mock.get('http://example.com/api/v2/event', exc=requests.exceptions.SSLError('test ssl'))
+        client = cls.BaseClient('http://example.com/api/v2/', ok_codes=(200, 201), verify=False)
+        with raises(requests.exceptions.SSLError, match="test ssl"):
+            client._http_request('get', 'event', resp_type='response')
```

Comment:
can we check here that the string "try selectin 'Trust any certificate'...' is not in the exception message?

Reply:
updated the match string, now it will test for an exact match.